src/databricks/labs/ucx/contexts/account_cli.py

```diff
@@ -19,6 +18,14 @@ def __init__(self, ac: AccountClient, named_parameters: dict[str, str] | None =
     def account_client(self) -> AccountClient:
         return self._ac
 
+    @cached_property
+    def workspace_client(self) -> WorkspaceClient:
```

Comment:
Updating the `assign-metastore` command to create the UCX catalog requires a `workspace_client` next to the `AccountClient` in the command. This is a bit of a hack to get __a__ `WorkspaceClient`

Linking issue to track how to do this properly

Reply:
#2507